zerver/views/message_edit.py

```diff
@@ -99,6 +107,7 @@ def get_message_edit_history(
     user_profile: UserProfile,
     *,
     message_id: PathOnly[NonNegativeInt],
+    allow_empty_topic_name: Json[bool] = False,
```

Comment:
We might be able to skip adding this parameter. I suspect that no client app uses this API endpoint -- the only thing they use is the edit history data that comes in `GET /messages` responses and events.

I guess the benefit to doing so is that that we'd save ourselves a future transition of changing the default from False => True here.

Reply:
>  I suspect that no client app uses this API endpoint

Terminal uses this endpoint.

See: https://chat.zulip.org/#narrow/channel/378-api-design/topic/edit.20history/near/1933192

Reply:
OK, thanks for checking, nothing can be readily improved here, then.